assets/konnectivity/konnectivity-server-services.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: konnectivity-server
+  labels:
+    app: kube-apiserver
+spec:
+  type: NodePort
+  selector:
+    app: kube-apiserver
+  ports:
+    - port: {{ .KonnectivityAgentPort }}
+      targetPort: {{ .KonnectivityAgentPort }}
```

Comment:
Shouldn't this be the server port not the agent port?

Reply:
See above.